hitchhiker/cli/modules/list.py

```diff
@@ -14,8 +14,26 @@
 )
 @click.pass_context
 def list_cmd(ctx: click.Context, glob: str) -> None:
-    """list all odoo modules"""
+    """
+    Lists all Odoo modules based on the provided glob.
 
+    Parameters:
+        ctx (click.Context): The Click context object.
+        glob (str): The glob pattern to search for Odoo modules.
+
+    Returns:
+        None
+
+    Description:
+    This function lists all Odoo modules based on the provided glob pattern.
+    It prints the module names and versions in a formatted table.
+
+    Example:
```

Comment:
There is no example needed. This is a click help text

Another idea would be to provide an example not as usage from code but from the command line